app/models/solid_queue/queue.rb

```diff
@@ -40,6 +40,19 @@ def size
       @size ||= ReadyExecution.queued_as(name).count
     end
 
+    def latency
+      @latency = begin
```

Comment:
Was this perhaps intended to be
```ruby
@latency ||= begin
```

Reply:
ooh good catch! 🙈

Reply:
I just fixed it and created a test for the memoization!